README.md

```diff
@@ -10,6 +10,7 @@ An alarm clock for Emacs
 
 -   Emacs 24 or higher
 -   [mpg123](http://mpg123.org) (gnu/linux)
+-   [mpg123](http://mpg123.org) (windows)
```

Comment:
I think it is better to merge L12 & L13 by modifying the '(gnu/linux)' to '(gnu/linux & Windows)'